### ISSUE TITLE
Contribute on the wiki section

### ISSUE THREAD

**Issue by shahriarhossain** [shahriarhossain/MailChimp.Api.Net#25]
@enkodellc : Thanks for the PR , have pushed it to nuget https://github.com/shahriarhossain/MailChimp.Api.Net/releases/tag/1.2.0.8

It would be awsome if you could post template related code sample on the wiki https://github.com/shahriarhossain/MailChimp.Api.Net/wiki

**Comment by enkodellc** [comment]
Yes.  I actually plan on submitting a pr with a web form with all your
wiki how to so someone has them all in one place to test all functionality
quickly might not be done  to  next  week.

**Comment by shahriarhossain** [comment]
@enkodellc : that would be awesome !

**Comment by enkodellc** [comment]
Feel free to assign this Issue to me. Like I said should be able to update the wiki next week and add additional functionality as well as a web form to utilize most of the API functionality. I will send you a link to the web form for review before I submit the PR. Thanks.

**Comment by shahriarhossain** [comment]
@enkodellc : I tried, don't know why but github can't find your usename, please assign yourself. 
![capture](https://cloud.githubusercontent.com/assets/6463320/16296933/4592b80c-38e3-11e6-837b-635e061c2561.PNG)

**Comment by enkodellc** [comment]
Strange, Maybe I need to be added as a contributor to the repo as  to utilize that functionality. https://help.github.com/articles/inviting-collaborators-to-a-personal-repository/

**Comment by enkodellc** [comment]
@shahriarhossain I am doing my best to review the code and add missing methods and clean up some of the code as I need them for my project... which is most of the functionality. Once this is ready I will have a more comprehensive view of the code and be better able to write up the wiki on how it all works. I just didn't want to write a bunch of wiki and then have to re-write it.
